docs/pages/ui-programming/user-interface-libraries.mdx

```diff
@@ -20,12 +20,6 @@ Below are some of the most popular UI Libraries. Test them all out and see what
 | [Gluestack UI](https://ui.gluestack.io/)                                          | [Link](https://ui.gluestack.io/)                                                                                |
 | [Ignite UI](https://docs.infinite.red/ignite-cli/)                                | [Link](https://docs.infinite.red/ignite-cli/boilerplate/components/)                            |
 | [React Native Reusables](https://github.com/mrzachnugent/react-native-reusables)  | [Link](https://github.com/mrzachnugent/react-native-reusables)                            |
+| [React Native UI Kitten](https://github.com/akveo/react-native-ui-kitten)         | [Link](https://akveo.github.io/react-native-ui-kitten/docs/getting-started/what-is-ui-kitten#what-is-ui-kitten) |
 
 > As with any library, it's always best to find one with a large and active community. This way, help is easy to come by. Check out any project's GitHub page to see what the activity is like.
```

Comment:
```suggestion
When selecting a library, select one that is supported by a large and active community. Check out a library's GitHub page to see its recent activity.
```

- This doesn't have to be a callout
- Let's be concise here in the message about choosing a library

Reply:
This note was not edited/added in this changeset, but let's merge it in. 🙂

Reply:
Oh my bad! Thanks for catching that!